### DISCUSSION TITLE
Export "AuthContext" and "useAuth" in "payload/auth"

### DISCUSSION THREAD

**Discussion by finkinfridom** [payloadcms/payload#1249] [Q&A] [answered]
I am playing around with the admin.avatar configuration and it was very difficult to spot the proper location for `useAuth` (similarly to what happens here: https://github.com/payloadcms/payload/blob/master/src/admin/components/graphics/Account/Gravatar.tsx).

I was able to import `useAuth` from `payload/dist/admin/components/utilities/Auth` but it's pretty nested.

Would it be possible to move the `useAuth` into `payload/auth`?

Thanks

**Comment by jacobsfletch** [comment, marked as answer]
@finkinfridom absolutely! It's in our roadmap to significantly improve our component library, including how components are exported as you mentioned, and also documentation. Here's a related discussion: https://github.com/payloadcms/payload/discussions/1729

If you need this right away we'd be happy to accept a PR!
